### PR TITLE
[86bxnqcn7][tooltip] added hint and information dropdown

### PR DESCRIPTION
**Pull request by msereniti**
## Motivation and Context

To make developers easier build accessible Tooltips UX, new components were added. `Hint` and `InforamtionDropdown` are reusing tooltip, but also are applying some special aria attributes to make it more accessible in their specific cases. 

See added documentation for more details.

## How has this been tested?

New unit tests + manual testing.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue).
- [x] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Nice improve.

## Checklist:

- [x] My code follows the code style of this project.
- [x] I have updated the documentation accordingly or it's not required.
- [x] Unit tests are not broken.
- [x] I have added changelog note to corresponding `CHANGELOG.md` file with planned publish date.
- [x] I have added new unit tests on added of fixed functionality.

### CODE REVIEW

**Comment by j-mnizhek** [comment]
Great changes! I'll check documentation right away.
BTW, should we add something to API page? Now I cannot know about the Hint and InformationDropdown components inside the Tooltip.
image

**Review by j-mnizhek — APPROVED** [review]

File: semcore/tooltip/src/Tooltip.jsx [hunk 1]
```diff
@@ -137,7 +139,103 @@ function TooltipPopper(props) {
...
+  static displayName = 'InformationDropdown';
+  static style = style;
+  static enhance = [uniqueIDEnhancement(), resolveColorEnhance()];
+  static defaultProps = {
+    ...defaultProps,
+    liveRegion: false,
+    interaction: 'click',
+  };
+  popperRef = React.createRef();
+  subcomponents = [InformationDropdown.Trigger.displayName, InformationDropdown.Popper.displayName];
```
[the remark this answers is not in the document]
Comment (msereniti) [reply semcore/tooltip/src/Tooltip.jsx]: Renamed to `DescriptionTooltip`.

File: website/docs/components/tooltip/examples/basic_usage.tsx [hunk 2]
```diff
@@ -1,16 +1,36 @@
...
+    <Tooltip>
+      <Tooltip.Trigger tag={Link}>Tooltip</Tooltip.Trigger>
+      <Tooltip.Popper>
+        Tooltip may contain icons, <Link href='https://semrush.com'>links</Link> and other elements.
+        On the other hand, it's content shouldn't be too long.
+      </Tooltip.Popper>
+    </Tooltip>
+    <InformationDropdown>
+      <InformationDropdown.Trigger tag={Link}>Information dropdown</InformationDropdown.Trigger>
+      <InformationDropdown.Popper>
```
[the remark this answers is not in the document]
Comment (msereniti) [reply website/docs/components/tooltip/examples/basic_usage.tsx]: My fault – the information dropdown tooltip trigger should handle space/enter keypresses, like interactive icons or buttons do. If developers need, they may teach a link to do it but to keep the example simple I've replaced the link with the button.

File: semcore/tooltip/src/Tooltip.jsx [hunk 3]
```diff
@@ -137,7 +139,103 @@ function TooltipPopper(props) {
...
+      </Hint.Trigger>
+      <Hint.Popper>{title}</Hint.Popper>
+    </>
+  );
+  handlePopperVisibleChange = (visible) => {
+    if (visible) {
+      setTimeout(() => {
+        this.popperRef.current.focus();
+      }, 0);
+    }
```
Comment (ilyabrower) [inline semcore/tooltip/src/Tooltip.jsx:204]: why not in `uncontrolledProps` ?

File: tools/playground/src/App.tsx [hunk 4]
```diff
@@ -58,13 +58,13 @@ export const PlaygroundWrapper: React.FC = () => {
...
-      </div>
-    </StrictMode>
+    // <StrictMode>
+    <div>
+      <Nav />
+      <main className='intergalactic-playground_playground-view'>
+        <PlaygroundWrapper />
+      </main>
+    </div>
+    // </StrictMode>
```
Comment (ilyabrower) [inline tools/playground/src/App.tsx:68]: you don't need this

File: semcore/tooltip/src/Tooltip.jsx (hunk 3 above)
Comment (msereniti) [reply semcore/tooltip/src/Tooltip.jsx:204]: Cause I don't want to additionally store  `visible` in the DescriptionTooltip. It doesn't work perfect and seems a to be risky a bit

File: tools/playground/src/App.tsx (hunk 4 above)
Comment (msereniti) [reply tools/playground/src/App.tsx]: Fixed

**Review by ilyabrower — APPROVED** [review]

**Pull request merged**